plugins/kafka/README.md

```diff
@@ -0,0 +1,175 @@
+#ECL Apache Kafka Plugin
+
+This is the ECL plugin to access [Apache Kafka](https://kafka.apache.org), a publish-subscribe messaging system.  ECL string data can be both published to and consumed from Apache Kafka brokers.
+
+Client access is via a third-party C++ plugin, [librdkafka](https://github.com/edenhill/librdkafka).
+
+##Installation and Dependencies
+
+[librdkafka](https://github.com/edenhill/librdkafka) is included as a git submodule in HPCC-Platform.  It will be built and integrated automatically when you build the HPCC-Platform project.
+
+The recommended method for obtaining Apache Kafka is via [source code](https://kafka.apache.org/documentation.html#quickstart), which also contains excellent instructions for setting up and interacting with topics.  Note that Apache Kafka has its own set of dependencies, most notably [zookeeper](https://zookeeper.apache.org).
```

Comment:
Good points, though I'm not sure about pointing the user toward the internal test code.  I reworded this section and then added complete ECL example code near the bottom of the README.  Disclosure:  I also fixed some grammar issues and added a paragraph regarding message distribution within Kafka.